style.css

```diff
@@ -533,9 +533,158 @@
 }
 
 .about-section {
+  margin-top: 120px;
   background-image: url(./images/Micro/1660951134783.png);
   width: 375px;
   height: 1901px;
   background-repeat: no-repeat;
   background-size: contain;
 }
+
+.contact-section {
+  margin-left: 24px;
+  margin-right: 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.contact-message {
+  margin-top: 120px;
+  margin-bottom: 58px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  max-width: 327px;
+}
+
+.contact-form-section {
+  list-style: none;
+  align-items: center;
+}
+
+#name {
+  margin: 15px 16px;
+  width: 295px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  border-style: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding-top: 14px;
+  padding-bottom: 14px;
+}
+
+#name::placeholder {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+#mail {
+  margin: 15px 16px;
+  width: 295px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  border-style: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding-top: 14px;
+  padding-bottom: 14px;
+}
+
+#mail::placeholder {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+#msg {
+  margin-top: 24px;
+  border-bottom: 24px;
+  width: 327px;
+  height: 114px;
+  padding: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background: #fbf8f7;
+  border-style: none;
+}
+
+#msg::placeholder {
+  font-family: "Inter", sans-serif;
```

Comment:
- [Optional] Consider creating a class for the font family rather than repeating the same property in all your styling. YOu can create a class for the font family and add that class to all the HTML elements that use that font. It is best practice to have a [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

-[Optional] It's better to follow commit message standards by making them capitalized and in the imperative form (Example: 'Modify readme file' instead of 'modified readme'). This way, your commit history will be more readable and more professional ✨